### PR TITLE
replace 'mark-as-deleted', hide and adhDeleteAction with adhHideAction

### LINKED ISSUE #2268: adhResourceAction.resourceWidgetDelete does not work
We currently have 2 places where the delete resource action is used:

-  mercator2015 proposal
-  mercator2016 proposal

Unfortunately, this action only works with resourceWidgets. We decided to deprecate resourceWidgets in #1711 and converted comments in #1743. mercator2016 was added in #1863 and has never been a resourceWidget. So basically deletion is broken with these two.

### PR DESCRIPTION
**Pull request by der-john**
(continues #2390.)

Strongly related to #2374 (follow-up: introduce `DELETE`), and thus to #2336 (physical deletion).

The original aim of this was:
- to translate the popup saying 'Do you really want to delete this?' when you hit 'delete' on an IdeaCollection proposal.

While doing this, I noticed that hide/delete functions were wrapped in extremely similar ways (including the aforementioned question) in the following five places:
- IdeaCollection
- Alexanderplatz
- Blog
- Comment
- Mercator2015

@xi and I discussed the possibility to factor out this functionality. I suggested adding a `resourceAction` to replace `adhDeleteAction`, since HIDE should be used where 'mark-as-deleted' used to be used.

In this PR,
- a resourceAction called `adhHideAction` is implemented;
- the resourceAction called `adhDeleteAction` is removed - this fixes #2268.

The following use cases are replaced with adhHideAction:
- IdeaCollection
- Alexanderplatz
- Mercator2015
- Blog (#2365 replaced 'mark-as-deleted' with physical deletion. The use case for this was deleting badge assignments, not blog contents. Therefore @2e2a and I suggest to replace the delete action in Blog.ts with hide, even though I couldn't test it right now.)

The following use cases are left untouched:
- Comment - because the template of the hide action is different (it's an icon).

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts [hunk 1]
```diff
@@ -62,24 +63,6 @@ export var shareActionDirective = () => {
...
-            class: "@"
-        },
-        link: (scope, element, attrs, column : AdhMovingColumns.MovingColumnController) => {
-            scope.delete = () => {
-                column.$broadcast("triggerDelete", scope.resourcePath);
-            };
-        }
-    };
-};
-
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts]: Why did you remove this?

File: src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/IdeaCollection.ts [hunk 2]
```diff
@@ -100,7 +100,8 @@ export var proposalDetailColumnDirective = (
     adhPermissions : AdhPermissions.Service,
     adhTopLevelState : AdhTopLevelState.Service,
     $location : angular.ILocationService,
-    $window : Window
+    $window : Window,
+    $translate
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/IdeaCollection.ts]: Not needed here

File: src/mercator/mercator/static/js/Packages/Blog/Blog.ts [hunk 3]
```diff
@@ -72,15 +72,16 @@ export var bindPath = (
...
     adhHttp : AdhHttp.Service<any>,
     adhPermissions : AdhPermissions.Service,
     adhPreliminaryNames : AdhPreliminaryNames.Service,
     adhShowError,
     adhSubmitIfValid,
-    adhUploadImage
+    adhUploadImage,
+    $q: angular.IQService,
+    $window: angular.IWindowService,
+    $translate
```
Comment (xi) [inline src/mercator/mercator/static/js/Packages/Blog/Blog.ts]: angular services should be injected first

Also: `$translate` is not used.

**Comment by xi** [comment]
You should not remove the delete functionality from Blog. adding hide is fine, but delete is required.

**Comment by xi** [comment]
It is a bit hard to review this for me. Could you please rebase this into 2 pull requests: One adding `hideActionDirective` and another one implementing translation on top of that?

File: src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts (hunk 1 above)
Comment (der-john) [inline src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts]: Because it was only used for Mercator2015, where I replaced it with `adhHideAction`.

**Comment by der-john** [comment]
No problem.

**Comment by der-john** [comment]
I've split it up.
